src/core/server/base.js

```diff
@@ -179,19 +192,37 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
             </I18nProvider>
           );
 
+          const asyncConnectLoadState = store.getState().reduxAsyncConnect.loadState || {};
+
+          // Create a list of any apiErrors detected.
+          const apiErrors = Object.keys(asyncConnectLoadState)
```

Comment:
We don't have a great way to be able to test this part, pulling it out into a util function will require the passing around of a lot of pieces for it to work. Any thoughts?